cost-analyzer/templates/cost-analyzer-service-template.yaml

```diff
@@ -1,5 +1,6 @@
 {{- $nginxPort := .Values.service.targetPort | default 9090 -}}
 {{- $servicePort := .Values.service.port | default 9090 -}}
+{{- $nodePort := .Values.service.nodePort | default 32000 -}}
```

Comment:
I think supplying a `nodePort` is optional if `type: NodePort` -  Probably want to avoid specifically choosing a value. Maybe use this as the conditional:
```
{{- if (eq .Values.service.type "NodePort") }}
{{- if .Values.service.nodePort }}
nodePort: .Values.service.nodePort
{{- end }}
{{- end }}

Reply:
fixed it !!